<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDisplayWdg</class>
 <widget class="QWidget" name="ImageDisplayWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetHeader" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>720</width>
     <height>56</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>56</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widgetHeader
{
background-color: rgba(170, 225,170,255);
border-top-left-radius: 8px;
border-top-right-radius: 8px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>676</x>
      <y>12</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-image: url(:/icon/close_normal32.png);
}
QPushButton:hover
{
	border-image: url(:/icon/close_hover32.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>720</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(197, 197, 197,160);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>600</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font-size: 16px;
	font-family: &quot;微软雅黑&quot;;
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetBody" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>71</y>
     <width>720</width>
     <height>614</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(220, 255, 220,255);
</string>
   </property>
   <widget class="QCheckBox" name="chb_name">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>16</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-size: 10pt;
	font-weight: 400;
	font-family: &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_link">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>16</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-size: 10pt;
	font-weight: 400;
	font-family: &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>链接</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>56</y>
      <width>656</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetTail" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>685</y>
     <width>720</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widgetTail
{
background-color: rgba(220, 255, 220,255);
border-bottom-left-radius: 8px;
border-bottom-right-radius: 8px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_down">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 1px solid rgb(200, 200, 250);
	border-radius: 20px;
	font-size: 10pt;
	color: rgb(0,0,0);
	font-weight: 400;
	font-family: &quot;黑体&quot;;
}
QPushButton:hover
{
	border: 1px solid rgb(200, 200, 250);
	background-color: rgba(170,170,240,220);
	border-radius: 20px;
	font-size: 10pt;
	color: rgb(255,255,255);
	font-weight: 400;
	font-family: &quot;黑体&quot;;
}</string>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>700</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(x1:0,y1:0,x2:1,y2:0,stop:0 rgba(255,255,255,255),stop:0.2 rgba(0,0,0,180),stop:0.5 rgba(0,0,0,220),stop:0.8 rgba(0,0,0,180),stop:1 rgba(255,255,255,255));
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>24</y>
      <width>500</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar#progressBar
{
color: rgba(50, 50, 50, 180);
font-size: 12pt;
font-family: &quot;黑体&quot;;
font-weight: 400;
border: 1px solid rgba(200, 200, 250, 200);
border-radius: 8px;
}
QProgressBar#progressBar::chunk
{
border: 1px solid rgba(200, 200, 250, 250);
border-radius: 8px;
background-color: qlineargradient(x1: 0,y1: 0,x2: 1,y2: 0,stop: 0 #7f7fd5,stop:0.5 #86a8e7,stop: 1.0 #91eae4);
}
</string>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>720</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>widgetHeader</zorder>
  <zorder>widgetBody</zorder>
  <zorder>widgetTail</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
